apps/api/src/routes/settings.ts

```
@@ -3,10 +3,12 @@
 import { db, schema, getDatabaseStats } from '@dxlander/database';
 import { eq, and } from 'drizzle-orm';
 import { randomUUID } from 'crypto';
+import { and, eq } from 'drizzle-orm';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'and'. Consider to delete it or rename it.

'and' is defined here:


(lint/suspicious/noRedeclare)